ios/Handlers/ChangePinHandler.swift

```diff
@@ -0,0 +1,45 @@
+protocol PinConnectorToPinHandler: AnyObject {
+    func onChangePinCalled(completion: @escaping (Bool, NSError?) -> Void)
+}
+
+enum PINEntryMode {
+    case login
+    case registration
+    
+}
+
+class ChangePinHandler: NSObject {
```

Comment:
any reason why the class conforms to NSObject? is it because of ONGCHangePinDelegate?

Reply:
Yes, this is because of ONGCHangePinDelegate